src/uint/modular/runtime_mod.rs

```diff
@@ -194,3 +233,36 @@ impl<const LIMBS: usize> zeroize::Zeroize for DynResidue<LIMBS> {
         self.montgomery_form.zeroize()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::nlimbs;
+
+    const LIMBS: usize = nlimbs!(64);
+
+    #[test]
+    #[allow(deprecated)]
+    // Test that a valid modulus yields `DynResidueParams`
+    fn test_valid_modulus() {
+        let valid_modulus = Uint::<LIMBS>::from(3u8);
+
+        DynResidueParams::<LIMBS>::new_checked(&valid_modulus).unwrap();
+        DynResidueParams::<LIMBS>::new(&valid_modulus);
+    }
+
+    #[test]
+    #[should_panic]
```

Comment:
A small nitpick, but this method can be tested without making it panic, since the panic currently originates from inside the test, not from inside the method.

Reply:
Done.